src/main/java/cloud/fogbow/ras/core/processors/FulfilledProcessor.java

```diff
@@ -95,6 +95,10 @@ protected void processFulfilledOrder(Order order) throws UnexpectedException {
             try {
                 LocalCloudConnector localCloudConnector = (LocalCloudConnector) CloudConnectorFactory.getInstance().
```

Comment:
We don't need this casting, because it is guaranteed that the CloudConnector is a local one. Instead of this command, use:
LocalCloudConnector localCloudConnector = new LocalCloudConnector(cloudName);

Reply:
That's right, I will remove it.